cashctrl_ledger/ledger.py

```diff
@@ -77,11 +352,34 @@ def delete_price():
         """
         raise NotImplementedError
 
-    def delete_vat_code():
-        """
-        Not implemented yet
+    def delete_vat_code(self, code: str):
         """
-        raise NotImplementedError
+        Deletes a VAT code from the tax system via a POST request to the API.
```

Comment:
Description should be
```
Deletes a VAT code from the remote CashCtrl account.
```

Instead of 
```
Deletes a VAT code from the tax system via a POST request to the API.
```